policy/writeRecipes.yaml

```diff
@@ -40,3 +40,6 @@ FitsStorage:
     variance:
       <<: *lossyBasic
 
+  # Set the default
+  default:
+    <<: *default
```

Comment:
Oh, one quick thing.  This is fine, but you might think about whether each of the full-Exposure schemes should be named more explicitly with an `&` tag, and then `default` could refer to the full-Exposure scheme rather than moving `&default` around.